contracts/voting-verifier/src/msg.rs

```diff
@@ -61,7 +61,7 @@ pub enum QueryMsg {
     #[returns(Poll)]
     GetPoll { poll_id: PollID },
 
-    #[returns(Vec<(CrossChainId, bool)>)]
+    #[returns(Vec<(connection_router::state::CrossChainId, bool)>)]
```

Comment:
What's the reason for this change?

Reply:
trying to build the json schema showed that the unqualified name couldn't be resolved